projects/avodata/AvodataSolverTool.java

```diff
@@ -0,0 +1,142 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Arrays;
+import java.util.Scanner;
+
+import org.apache.commons.math3.analysis.function.Abs;
+import org.apache.commons.math3.linear.Array2DRowRealMatrix;
+import org.apache.commons.math3.linear.DecompositionSolver;
+import org.apache.commons.math3.linear.MatrixUtils;
+import org.apache.commons.math3.linear.RealMatrix;
+import org.apache.commons.math3.linear.RealVector;
+import org.apache.commons.math3.linear.SingularValueDecomposition;
+import org.apache.commons.math3.util.CombinatoricsUtils;
+
+/** To build: bazel build //projects/avodata:ls-alg_tool To run: bazel-bin/projects/avodata/ls-alg_tool */
```

Comment:
Please use a multiline comment. Starting with `/*` prevents the formatter from formatting the comment (which is good since you want to preserve the newlines).
```
/* This class reads matrix from csv file and calculates profession attributes using least squares.
 * To build:
 * bazel build //projects/avodata:ls-alg_tool
 * To run:
 * bazel-bin/projects/avodata/ls-alg_tool
 */
```